_posts/2020-06-24-intellij-quarkus-tools-1.0.0.adoc

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: 'Quarkus Tools for IntelliJ 1.0.0 release !'
+date: 2020-06-24
+tags: announcement ide
+author: jeffmaury
+---
+:imagesdir: /assets/images/posts/intellij-quarkus-tools-1.0.0
+
+We are very pleased to announce the 1.0.0 release of Quarkus Tools for IntelliJ.
+This release brings enhanced code completion and syntax validation for `application.properties` and better support for Microprofile in Java files.
```

Comment:
could we mention this removes the dependency on the LSP plugin and thus considered much more stable ?